Atomic/util.py

```diff
@@ -215,3 +215,13 @@ def get_scanners():
             except AttributeError:
                 pass
     return scanners
+
+def default_docker():
+    atomic_config = get_atomic_config()
+    default_docker = atomic_config.get('default_docker')
+    if not default_docker:
+        default_docker = "docker"
+    return default_docker
```

Comment:
should we somehow validate (like in_array(valid_dockers_array_strings) the `default_docker` string got from the config and have a nicer error? not issue otherwise